jabdul/crud-api
# Feature/upgrade hapi

Bumped  Hapi version to v20. Bumped stale dependencies to latest version and replaced broken packages swagger package to a more maintained [hapi-swagger.](https://github.com/glennjones/hapi-swagger)

To test, complete the following steps
- [ ] yarn 
- [ ]  yarn test
- [ ] yarn bootstrap

Check **/documentation**  route to confirm that swagger docs are displayed.

PR is based off feature/upgrade-mongoose
